orangecontrib/text/widgets/ownyt.py

```diff
@@ -30,12 +30,16 @@ class OWNYT(OWWidget):
 
     outputs = [(Output.CORPUS, Corpus)]
     want_main_area = False
+    resizing_enabled = False
+
+    QT_DATE_FORMAT = 'yyyy-MM-dd'
+    PY_DATE_FORMAT = '%Y-%m-%d'
 
     # Settings.
     recent_queries = Setting([])
     recent_api_keys = Setting([])
-    year_from = Setting("")
-    year_to = Setting("")
+    date_from = Setting(datetime(2000, 1, 1).date().strftime(PY_DATE_FORMAT))
```

Comment:
Didn't work because it wasn't set to. Please try again. I also set the initial date range to 1 year.